docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<blockquote>&gt;&nbsp;Formally, comparing two items&nbsp;&nbsp;i and&nbsp;&nbsp;j that differ by an amount&nbsp;&nbsp;x&nbsp;on predictor</blockquote><p>&nbsp;I would choose something different than "x" since "x" is also used for the predictors. The concepts you're outlining are perfect, I would only change the letters in the mentioned case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>